fabbwled-backend/src/main/resources/application.yaml

```diff
@@ -23,4 +23,4 @@ spring:
       enabled: ALWAYS
 springdoc:
   api-docs:
-    path: "/api-docs"
\ No newline at end of file
+    path: "/api-docs"
```

Comment:
note that the current `.editorconfig` states `insert_final_newline = false`
while I am not a fan of this setting, that's currently the contract. if you deliberately want to update it, also reflect it in the `.editorconfig` file.